apps/els_lsp/src/els_sup.erl

```diff
@@ -79,6 +79,10 @@ init([]) ->
                , #{ id       => els_server
                   , start    => {els_server, start_link, []}
                   }
+               , #{ id       => els_log_notification
```

Comment:
Do we really need a process for this? Can't we just add the handler at boot?

Reply:
My reasoning was that the handler needs to start after els_server. The start function actually returns `ignore` so there is no process.

But you are right, I can just statically add the handler in `erlang_ls.erl`